### ISSUE TITLE
Need Sponsor Logos on Session Modals and Posters

### ISSUE THREAD

**Issue by dzoladz** [alaoweb/2020.alaoweb.org#90]
In order to get the sponsor logos on the website before we make the transition from WA to Jekyll, I'm closing #40 with a recently added PR and opening this as a separate issue.

**Comment by kenirwin** [comment]
Do we have information about who the sponsors are for the poster sessions? (Are the posters sponsored by companies, in the sense that they give us money and we put their logo on the poster session pages?)

And am I right in thinking that the session sponsors will typically be ALAO interest groups and such? Are there other kinds of sponsors for long sessions?

**Comment by dzoladz** [comment]
- **Gold Sponsors** will have their logo on all session modals, and all posters.
- **Sessions Sponsors** will have their logo on all session modals.
- **Posters Sponsors** will have their logo on all posters.
- **Individual Session Sponsors** play $150 per session. Since we don't have sessions to chose from at the moment, we're still waiting on the placement of those logos.

I'll need to reach out to Sara about the IG roles.

**Comment by dzoladz** [comment]
Most of the work on this issue has been completed. If we need to address something more specific, please create a new issue with the details for us to address.
